### ISSUE TITLE
Printable and shareable and readable feeds

### ISSUE THREAD

**Issue by GreenLunar** [lwindolf/liferea#1086]
Not only it makes RSS pages to look better, it also makes them printable, and also exportable to PDF with hyperlinks too.  See attachments set

Test link: https://archlinuxarm.org/forum/feed.php
a) [screenshot](https://github.com/OtterBrowser/otter-browser/files/8014673/Arch.Linux.ARM.png.pdf)
b) [shareable in pdf and hyperlinks](https://github.com/OtterBrowser/otter-browser/files/8014680/Arch.Linux.ARM.pdf)
c) [without xsl](https://github.com/OtterBrowser/otter-browser/files/8014724/Arch.Linux.ARM.otter.pdf)

https://github.com/QupZilla/qupzilla/issues/1361#issuecomment-262802841

**Comment by lwindolf** [comment]
You mean we should provide a CSS for media target printer?

**Comment by sjehuda** [comment]
Yes, namely for screen display. I think printer display/ccss can remain simple.

**Comment by sjehuda** [comment]
Can we reopen this please?

**Comment by lwindolf** [comment]
Printing was implemented recently in 1.16. See menu "Tools" -> "Print"
